### DISCUSSION TITLE
Dark mode?

### DISCUSSION THREAD

**Discussion by DrMan1209** [AntonyCorbett/OnlyM#479] [Ideas]
I would love if the OnlyM app had a dark mode. This doesn't seem like a monumental change as, for the most part, you are changing the background to black and text to white.
The reason I open this discussion is to try to understand the codebase so I could implement it myself.
How would I do it, where do I start and would that be possible?

**Comment by AntonyCorbett** [comment]
@DrMan1209 Good question!

I use the following MaterialDesign library that dictates the appearance of UI elements:

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit

Simple changes to OnlyT's App.xaml file can modify the colours. For example, if we change this line:

`<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Light.xaml" />
`

to this:

`<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Dark.xaml" />`

we get a change to dark mode as shown below (although you would probably want to modify the colour scheme used too (currently DeepPurple and Lime) to make for better contrast.

![image](https://github.com/AntonyCorbett/OnlyM/assets/24551146/fad50487-dbc5-4175-afc5-252d080d2de0)

**Comment by AntonyCorbett** [comment]
@DrMan1209 available in latest pre-release
